src/pages/AdminPage.jsx

```diff
@@ -293,8 +118,27 @@ const AdminPage = () => {
   const [isDateAscend, setDateAscend] = useState(false);
   const [isVolunteerAscend, setVolunteerAscend] = useState(false);
   const [isApprovalAscend, setApprovalAscend] = useState(false);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [data, setData] = useState(dummy);
+
+  const m = new Map();
+  for (let i = 0; i < data.length; i += 1) {
+    m.set(data[i].id, false);
+  }
+
+  const [checked, setChecked] = useState(m);
+
+  const getSpecies = async () => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'getSpecies' is assigned a value but never used.